CombineUnloadAIDriver.lua

```diff
@@ -1805,7 +1805,7 @@ function CombineUnloadAIDriver:unloadMovingCombine()
 	end
 
 	--when the combine is empty, stop and wait for next combine
-	if self:getCombinesFillLevelPercent() <= 0.1 then
+	if self:getCombinesFillLevelPercent() <= 0.1 and self.vehicle.cp.followAtFillLevel >0 then
```

Comment:
@schwiti6190 refactored the settings in the meanwhile, vehicle.cp.followAtFillLevel is now an object, use vehicle.cp.settings.followAtFillLevel:get() (or self:getFillLevelThreshold()

Reply:
thangs for the update